src/app/item-popup/ItemDescription.m.scss

```diff
@@ -7,13 +7,11 @@
 
 .officialDescription {
   composes: description;
-  overflow: auto;
```

Comment:
These child elements having overflow also triggered scroll chaining, and I cannot see why they every needed it.